masterlist.yaml

```diff
@@ -125,6 +122,8 @@ plugins:
   - name: 'Anchorage.esm'
     url: [ 'http://taleoftwowastelands.com/']
     priority: -400998000
+    after:
+      - 'Fallout3.esm'
```

Comment:
Yes, It have.

TaleOfTwoWastelands.esm have them all as masters and in correct order.

I've pointed it explicitly to remove possible collisions.